aries-site/src/components/home/Quote.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Avatar, Box, Image, Paragraph, Stack, Text } from 'grommet';
+import { useDarkMode } from '../../utils';
+
+export const Quote = () => {
+  const darkMode = useDarkMode();
+
+  const quoteLines = `/static/images/quote-lines${
+    darkMode.value ? '-dark' : ''
+  }.svg`;
+
+  return (
+    <Box direction="row" gap="large" justify="center" pad="large">
+      <Box
+        align="center"
+        justify="center"
+        gap="medium"
+        background={{
+          image: `url(${quoteLines})`,
+          size: 'auto',
+          position: 'top left',
+        }}
+        pad={{ left: 'xlarge' }}
+        flex={false}
+      >
+        <Avatar src="/static/images/t_harms_bio_image.png" size="149px" />
+        <Box align="center">
+          <Text size="xlarge" weight="bold" color="text-strong">
+            Tim Harms
+          </Text>
+          <Text size="xlarge">HPE Head of Brand</Text>
+        </Box>
+      </Box>
+      <Stack>
+        <Box
+          border={{ size: '3px', color: 'text-weak' }}
+          pad="medium"
+          margin="small"
+          width={{ max: '678px' }}
+        >
+          <Paragraph size="xxlarge" textAlign="center">
+            The HPE Design System empowers developers, designers and others to
+            create consistent, accessible and flexible interfaces
+          </Paragraph>
+        </Box>
+        <Box fill align="end" justify="end">
+          <Image src={quoteLines} margin={{ right: '-138px' }} />
+        </Box>
+        <Box fill justify="between">
+          <Text
+            size="124px"
```

Comment:
Any grommet text size that would get us close here?

Reply:
Don't think so. For comparision, xxlarge is 36px.

Reply:
What about `<Text size="6xl">`?

Reply:
Interesting. Didn't realize that was a thing for Text. 6xl comes out to 90px which is still a bit small compared to the figma. Looks something like this (top left is 6xl, bottom right is still 124px): 
![image](https://user-images.githubusercontent.com/12054362/107086369-24435280-67b7-11eb-8266-85788180d685.png)

Reply:
BTW, the background lines in the quote were a little tricky but this seems to resize pretty nicely I think.

Reply:
I get why the quote area would shrink to the right of the image, any way we could have it stack on the bottom of the image when it's down to the smaller screens?

Reply:
Yup. Done.

Reply:
Did we opt against the `6xl` here?

Reply:
@vavalos5 Thoughts? As you can see from the above image `6xl` is significantly smaller (top left quote) than the figma design (bottom right quote).

Reply:
That's fine, I had these on Figma as h1 xlarge, but "6xl" is fine

Reply:
From a screenreader standpoint I don't think h1 would be ok here. I'll use 6xl if you're good with it.

Reply:
Agreed!